Example/SmileID/Home/HomeViewModel.swift

```diff
@@ -72,6 +100,23 @@ class HomeViewModel: ObservableObject,
             jobName: "SmartSelfie Authentication",
             apiResponse: apiResponse
         )
+        if let apiResponse = apiResponse {
+            dataStoreClient.saveJob(
+                data: JobData(
+                    jobType: .smartSelfieAuthentication,
+                    timestamp: apiResponse.createdAt,
+                    userId: apiResponse.userId,
+                    jobId: apiResponse.jobId,
+                    jobComplete: true,
```

Comment:
This should add a check for whether the job status is pending or not. i.e. it might be pending if it went to human review

Reply:
I got the inspiration from the Android project - https://github.com/smileidentity/android/blob/8a5324be2d3f167e4f8bea2dad42e42a54117314/sample/src/main/java/com/smileidentity/sample/viewmodel/MainScreenViewModel.kt#L261

Does that mean when the status is also not rejected, cause there are 3 possible values for the status or should it be set to true if status is approved?